### PR TITLE
Add SDSU to Toyota/Lexus DBCs

### PR DESCRIPTION
**Pull request by ErichMoraga**

### CODE REVIEW

**Comment by adeebshihadeh** [comment]
Can you make a new DBC file like the comma one called _community.dbc?

**Comment by adeebshihadeh** [comment]
The relevant DBCs need to be updated to include the community DBC.

File: generator/toyota/toyota_nodsu_hybrid_pt.dbc [hunk 1]
```diff
@@ -1,6 +1,8 @@
 CM_ "IMPORT _toyota_2017.dbc";
 CM_ "IMPORT _comma.dbc";
+CM_ "IMPORT _toyota_nodsu_bsm.dbc";
```
Comment (adeebshihadeh) [inline generator/toyota/toyota_nodsu_hybrid_pt.dbc:3]: remove this

File: generator/toyota/toyota_nodsu_pt.dbc [hunk 2]
```diff
@@ -1,6 +1,21 @@
 CM_ "IMPORT _toyota_2017.dbc";
 CM_ "IMPORT _comma.dbc";
+CM_ "IMPORT _toyota_nodsu_bsm.dbc";
```
Comment (adeebshihadeh) [inline generator/toyota/toyota_nodsu_pt.dbc:3]: remove this

File: generator/toyota/toyota_nodsu_pt.dbc [hunk 3]
```diff
@@ -1,6 +1,20 @@
 CM_ "IMPORT _toyota_2017.dbc";
 CM_ "IMPORT _comma.dbc";
 CM_ "IMPORT _toyota_nodsu_common.dbc";
+CM_ "IMPORT _community.dbc";
+
+BO_ 401 STEERING_LTA: 8 XXX
```
Comment (adeebshihadeh) [inline generator/toyota/toyota_nodsu_pt.dbc:6]: remove this msg

**Comment by ErichMoraga** [comment]
Weird.  Latest check fails...
```
scons: *** [opendbc/can/dbc_out/toyota_nodsu_pt_generated.cc] Explicit exit, status Duplicate address detected 401 toyota_nodsu_pt_generated
scons: building terminated because of errors.
Duplicate address detected 401 toyota_nodsu_pt_generated
The command '/bin/sh -c scons -c && scons -j$(nproc)' returned a non-zero code: 1

Error: Process completed with exit code 1.
```
...yet I removed 401 from generator/toyota/toyota_nodsu_pt.dbc, and there are no instances of it there anymore.

**Review by ErichMoraga — COMMENTED** [review]
Needed to do the ICE version instead, to get the latest check failure to pass...
```
scons: *** [opendbc/can/dbc_out/toyota_nodsu_pt_generated.cc] Explicit exit, status Duplicate address detected 401 toyota_nodsu_pt_generated
scons: building terminated because of errors.
Duplicate address detected 401 toyota_nodsu_pt_generated
The command '/bin/sh -c scons -c && scons -j$(nproc)' returned a non-zero code: 1

Error: Process completed with exit code 1.
```

...though I may need to remove 401 from the hybrid version too.  Will wait 'n see.

**Comment by adeebshihadeh** [comment]
Converted to draft until CI passes

**Comment by adeebshihadeh** [comment]
Closing due to inactivity. Feel free to re-open when ready.

**Pull request closed without merging**